### PR TITLE
Random default group images

### PR DESCRIPTION
**Pull request by gdpelican**

### CODE REVIEW

File: app/models/default_group_cover.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class DefaultGroupCover < ActiveRecord::Base
+  has_attached_file :cover_photo
+  validates_attachment :cover_photo, content_type: { content_type: ["image/jpg", "image/jpeg", "image/png", "image/gif"] }
+
+  def self.sample
+    all.order('random()').first
+  end
+
+  def self.store(file)
+    file = open(file) if file.is_a? String
```
Comment (robguthrie) [inline app/models/default_group_cover.rb]: is this a IO open? If so you might need to close it. Also if so, use a do |file| end so you don't need to close it manually.

File: lineman/app/models/group_records_interface.coffee [hunk 2]
```diff
@@ -6,3 +6,6 @@ angular.module('loomioApp').factory 'GroupRecordsInterface', (BaseRecordsInterfa
       @fetch
         path: "#{parentGroup.id}/subgroups"
         cacheKey: "subgroupsFor#{parentGroup.key}"
+
+    uploadPhoto: (group, file, kind) =>
```
Comment (robguthrie) [inline lineman/app/models/group_records_interface.coffee]: I think you can drop onSuccess, onFailure

Comment (gdpelican) [inline lineman/app/models/group_records_interface.coffee]: Cool, this is a merge artifact I think; the uploadPhoto method should be on group_model now.

File: app/models/default_group_cover.rb (hunk 1 above)
Comment (gdpelican) [inline app/models/default_group_cover.rb]: Yep, it's open-uri; made that change to use a block.

**Comment by gdpelican** [comment]
The merge :monkey: strikes!

**Pull request merged**